backend/tests/ohq/test_commands.py

```diff
@@ -596,7 +593,10 @@ def test_questions_per_ta_computation(self):
         yesterday = timezone.datetime.today().date() - timezone.timedelta(days=1)
         yesterday_weekday = yesterday.weekday()
 
-        expected_8 = self.ta_1_questions_8 / self.num_tas_8
+        expected_8 = (
+            self.ta_1_questions_8_week_1 / self.num_tas_8
```

Comment:
Can we modify this test case to catch possible regression where self.num_tas_8 > 1?